README.md

```
@@ -81,13 +81,13 @@ jobs:
 ### `message`
 
 - **Description**: check commit message formatting convention.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)